src/Domain/Tags.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines tags related to domain quantities
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/DataBoxTag.hpp"
+#include "DataStructures/Index.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/Element.hpp"
+
+namespace Tags {
+
+/// The ::Element associated with the DataBox
+template <size_t VolumeDim>
+struct Element : db::DataBoxTag {
+  static constexpr db::DataBoxString_t label = "Element";
+  using type = ::Element<VolumeDim>;
+};
+
+/// The extents of DataVectors in the DataBox
+template <size_t VolumeDim>
+struct Extents : db::DataBoxTag {
+  static constexpr db::DataBoxString_t label = "Extents";
+  using type = ::Index<VolumeDim>;
+};
+
+/// The coordinate map from logical to grid coordinate
+template <size_t VolumeDim>
+struct ElementMap : db::DataBoxTag {
```

Comment:
I'm also now wondering if `ElementMap` things should go in the `dg` namespace. Opinions?

Reply:
again an `ElementMap` is generic and not restricted to DG